### ISSUE TITLE
Changing 'Timer.MAX_RUNTIME_PER_DAY' works weird.

### ISSUE THREAD

**Issue by newbieh4cker** [ericyd/gdrive-copy#139]
### Are you requesting a feature or reporting a bug?
bug, maybe

### If reporting a bug, is there already an issue open for this same bug?
not really

### What is the current behavior?
script stops after running about 2 hours

### What is the expected behavior?
script has to run about 6 hours

as #120 , I've changed some of time variables.
![image](https://user-images.githubusercontent.com/23703741/71386019-3b17fb80-262e-11ea-8b43-00d593040a4f.png)

If I understood right way, my script has to run about 6 hours(355min).
but If I see the the "Copy Folder Log~" file, it runs about 2 hours.
What's wrong??

**Comment by ericyd** [comment]
Hm. Well, I would have expected that to work. Are you sure your changes are deployed, and you're testing against the latest version of your code?

**Comment by newbieh4cker** [comment]
Yes I am, It is deployed and using latest version of deployed code
